testsupport/tiers/checks.go

```diff
@@ -271,6 +275,57 @@ func (a *testTierChecks) GetClusterObjectChecks() []clusterObjectsCheck {
 	return []clusterObjectsCheck{}
 }
 
+type appstudioTierChecks struct {
+	baseTierChecks
+}
+
+func (a *appstudioTierChecks) GetTierObjectChecks() []tierObjectCheck {
+	return []tierObjectCheck{nsTemplateTier(a.tierName, 30)}
+}
+
+func (a *appstudioTierChecks) GetNamespaceObjectChecks(nsType string) []namespaceObjectsCheck {
+	checks := append(commonChecks,
```

Comment:
you cannot use the `commonChecks` as they expect `user-edit` RoleBinding which is not part of this tier